docs/web_link.md

```diff
@@ -0,0 +1,49 @@
+Web Links
+=========
+
+Web links are objects that point to URLs. These objects are also known as
+bookmarks within the Box web application. Web link objects are treated
+similarly to file objects.
+
+Create Web Link
+---------------
+
+Calling `folder.create_web_link(target_url, name=None, description=None)` will let you create a new web link with a specified name and description.
+
+```python
+folder_id = '1234'
+target_url = 'https://example.com'
+link_name = 'Example Link'
+link_description = 'This is the description'
+folder = client.folder(folder_id).create_web_link(target_url, link_name, link_description)
```

Comment:
The variable on the left here should probably be named something like `web_link`